### PR TITLE
bugfix: Don't highlight generated given if it doesn't exist

### PR DESCRIPTION
**Pull request by tgodzik**

### CODE REVIEW

File: tests/cross/src/test/scala/tests/highlight/Scala3DocumentHighlightSuite.scala [hunk 1]
```diff
@@ -107,4 +107,28 @@ class Scala3DocumentHighlightSuite extends BaseDocumentHighlightSuite {
...
+    "given-not-synthetic1",
+    """|given <<`giv@@en_D`>>: Double = 4.0
+       |val a = <<`given_D`>>""".stripMargin,
+  )
+
+  check(
+    "given-not-synthetic2",
+    """|given <<`giv@@en_D`>>:Double = 4.0
+       |val a = <<`given_D`>>""".stripMargin,
+  )
```
Comment (jkciesluk) [inline tests/cross/src/test/scala/tests/highlight/Scala3DocumentHighlightSuite.scala:132]: the `@@` should be in `val a` ? It's in given in previous test

File: tests/cross/src/main/scala/tests/BaseDocumentHighlightSuite.scala [hunk 2]
```diff
@@ -27,7 +27,7 @@ class BaseDocumentHighlightSuite extends BasePCSuite with RangeReplace {
       val highlights = presentationCompiler
         .documentHighlight(
           CompilerOffsetParams(
-            URI.create("file:/Highlight.scala"),
+            URI.create("file:///Highlight.scala"),
```
Comment (jkciesluk) [inline tests/cross/src/main/scala/tests/BaseDocumentHighlightSuite.scala:30]: Just asking, why this change?

File: tests/cross/src/test/scala/tests/highlight/Scala3DocumentHighlightSuite.scala [hunk 3]
```diff
@@ -107,4 +107,28 @@ class Scala3DocumentHighlightSuite extends BaseDocumentHighlightSuite {
...
+
```
Comment (jkciesluk) [inline tests/cross/src/test/scala/tests/highlight/Scala3DocumentHighlightSuite.scala:133]: Could you also add test like
```scala
given Int = 10
val a = given_I
```

**Review by jkciesluk — CHANGES_REQUESTED** [review]
Looks good, I just have some questions about tests

File: tests/cross/src/test/scala/tests/highlight/Scala3DocumentHighlightSuite.scala (hunk 3 above)
Comment (tgodzik) [reply tests/cross/src/test/scala/tests/highlight/Scala3DocumentHighlightSuite.scala:145]: Done!

File: tests/cross/src/test/scala/tests/highlight/Scala3DocumentHighlightSuite.scala (hunk 1 above)
Comment (tgodzik) [reply tests/cross/src/test/scala/tests/highlight/Scala3DocumentHighlightSuite.scala]: Ach, good catch. Fixed

File: tests/cross/src/main/scala/tests/BaseDocumentHighlightSuite.scala (hunk 2 above)
Comment (tgodzik) [reply tests/cross/src/main/scala/tests/BaseDocumentHighlightSuite.scala]: Not needed, there are some warning popping up and thought that might fix it, but it didn't. The warnings are not relevant, just pop up in tests.

**Review by jkciesluk — APPROVED** [review]
LGTM
